apps/user/src/user/user.service.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@nestjs/common';
+import { RegisterUserDto } from '@dtos/register-user.dto';
+
+@Injectable()
+export class UserService {
+  async registerUser(registerUserDto: RegisterUserDto) {
+    const username = registerUserDto.username;
+    return { message: 'create user service', data: { username } };
```

Comment:
@dorukucak this is a e2e test, so I need you to do the real thing with Keycloak.